src/Data/List/Properties.agda

```diff
@@ -575,6 +576,43 @@ module _ {a} {A : Set a} where
   tabulate-lookup []       = refl
   tabulate-lookup (x ∷ xs) = P.cong (_ ∷_) (tabulate-lookup xs)
 
+  tabulate-len : ∀ {n} → (f : Fin n → A) →
+                 length (tabulate f) ≡ n
+  tabulate-len {zero} f = refl
+  tabulate-len {suc n} f = cong suc (tabulate-len (λ z → f (suc z)))
+
+  tabulate-len≤l : ∀{n} → (f : Fin n → A) →
```

Comment:
These two properties are immediate corollaries from the first properties and the reflexivity of `≤` no? I don't think they should be included.

Reply:
I used these with ```lookup'``` , so I didn't want to use ```subst```. Now they are not necessary.